Closr/RestaurantTableViewController.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  RestaurantTableViewController.swift
+//  TableViewTest
+//
+//  Created by Yale的Mac on 2017/2/27.
+//  Copyright © 2017年 YaleMac. All rights reserved.
+//
+
+import UIKit
+
+class RestaurantTableViewController: UITableViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(RestaurantTableViewCell.self, forCellReuseIdentifier: "reuseIdentifier")
+        
+        tableView.delegate=self
+        tableView.dataSource=self
+        
+        //navigationItem.title="TABLE"
+        
+        // self.view.addSubview(RestaurantTableViewController)
+        
+        if Device.is_3_5_inches || Device.is_4_inches {
```

Comment:
take a look at UITableViewAutomaticDimension so you don't need to define row height, auto layout will figure it out